docs/Developers/Git_Basics.rst

```diff
@@ -300,7 +300,7 @@ Now after you have created the Pull Request, there are two possibilities:
 - your PR will be rejected. There are 2 cases when a PR is rejected:
 
       - Test fails
-      - Reviewer wants something changed (This also causes gitmate to fail)
+      - The reviewer wants something changed (This also causes gitmate to fail)
```

Comment:
although adding `the` is technically grammatically correct, the impact that it has on clarity is minimal, so not sure if we need to add this.